defect-tracker-server/src/main/java/com/defect/tracker/services/DefectServiceImpl.java

```diff
@@ -49,4 +54,20 @@ public boolean isDefectExistsById(Long id) {
 		return defectRepository.existsById(id);
 	}
 
+	@Override
+	public DefectCountByPriotryAndProjectDto countByProject(String projectName) {
+		Project project = projectService.findByName(projectName);
+		DefectCountByPriotryAndProjectDto defectCountDto = new DefectCountByPriotryAndProjectDto();
+		defectCountDto.setPriorityHigh(defectRepository.countByPriorityAndProject("high", project));
```

Comment:
can use the project name directly

Reply:
you can use countByPriorityAndProjectName("high",projectName)

Reply:
face the error beacause getter and setter type is Project

Reply:
did you implement like this? defectCountDto.setPriorityHigh(defectRepository.countByPriorityAndProjectName("high", projectName));